src/display/editor/editor.js

```diff
@@ -507,15 +507,19 @@ class AnnotationEditor {
     }
   }
 
-  fixAndSetPosition() {
+  /**
+   * Fix the position of the editor in order to keep it inside its parent page.
+   * @param {number} rotation - the rotation of the page.
```

Comment:
Let's make sure that we mark this as optional.
```suggestion
   * @param {number} [rotation] - the rotation of the page.
```